ProfessorAssessor/View Controllers/SignUpViewController.m

```diff
@@ -75,18 +113,27 @@ - (void)didSelectSchool:(School *)school {
     [self dismissViewControllerAnimated:YES completion:nil];
 }
 
+- (void)saveFacebookAccountInformation {
```

Comment:
I see what you're doing here and it makes sense but better to have facebook login create a user and then just take them to the home screen.